package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@polymer/iron-location",
+  "private": true,
+  "description": "Bidirectional data binding into the page's URL.",
+  "repository": {
```

Comment:
Short proposal:

```
"repository": "PolymerElements/iron-location",
```

Reply:
This is just copied from bower. Doesn't seem to be any advantage to changing it.

https://docs.npmjs.com/files/package.json#repository